src/apps/dolomite/arbitrum/dolomite.borrow_positions.contract-position-fetcher.ts

```diff
@@ -17,6 +19,19 @@ export class ArbitrumDolomiteBorrowPositionsContractPositionFetcher extends Dolo
     super(appToolkit, dolomiteContractFactory);
   }
 
+  async getLabel(
+    params: GetDisplayPropsParams<DolomiteMargin, DolomiteDataProps, DolomiteContractPositionDefinition>,
+  ): Promise<string> {
+    // The label is determined based on the unique account struct that is associated with the position
+    const castedPosition = params.contractPosition as DolomiteContractPosition;
+    if (castedPosition.accountStructs && castedPosition.accountStructs.length === 1) {
```

Comment:
This `if` statement never executes because `params.contractPosition` is never an instance of `DolomiteContractPosition`.

The issue I'm facing is `getLabel` needs to be able to tell me which `account` we are operating on, to generate a label dynamically.

If that's not possible, we can just have no label at all (return an empty string, like the `getLabel` implementation for Dolomite Balances).